forge-gui/res/cardsfolder/upcoming/your_plans_mean_nothing.txt

```diff
@@ -0,0 +1,13 @@
+Name:Your Plans Mean Nothing
+ManaCost:no cost
+Types:Scheme
+T:Mode$ SetInMotion | ValidCard$ Card.Self | Execute$ DBChoosePlayer | TriggerZones$ Command | TriggerDescription$ When you set this scheme in motion, any number of target players each discard their hands. Each opponent draws cards equal to the number of cards that player discarded minus one. Then if you discarded a card this way, draw seven cards.
+SVar:DBChoosePlayer:DB$ Pump | ValidTgts$ Player | TargetMin$ 0 | TargetMax$ PlayerCount | TgtPrompt$ Choose any number of players | SubAbility$ DBRepeat
+SVar:DBRepeat:DB$ RepeatEach | RepeatPlayers$ Targeted | RepeatSubAbility$ DBDiscard | SubAbility$ DBYouDraw | SpellDescription$ Each opponent draws cards equal to the number of cards that player discarded minus one. Then if you discarded a card this way, draw seven cards.
+SVar:DBDiscard:DB$ Discard | Defined$ Remembered | Mode$ Hand | RememberDiscarded$ True | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | NumCards$ X | Defined$ Player.Opponent+IsRemembered | SubAbility$ DBClearRemembered
+SVar:DBYouDraw:DB$ Draw | Defined$ You | NumCards$ 7 | ConditionDefined$ Targeted | ConditionPresent$ You

```

Comment:
this condition isn't good enough in case your hand is empty
better filter X with `Valid Card.RememberedPlayerOwn` and only clean up at the very end instead

Reply:
I think I'm misunderstanding you. I have not been able to get this to work using examples from other cards. Could explain a little more?

Reply:
`SVar:X:Remembered$Valid Card.RememberedPlayerOwn/Minus.1`
+
`ConditionDefined$ Remembered | ConditionPresent$ Card.YouOwn`

Reply:
Haha I just finished adding that and am testing it. That always happens when you ask questions 😭 . Found it on [[Fanatic of the Harrowing]]